tst/utils/Thread.cpp

```diff
@@ -183,13 +184,19 @@ TEST_F(ThreadFunctionalityTest, ThreadCreateWithParamsNegativeTest)
     ThreadParams threadParams;
     threadParams.version = 0;
     threadParams.stackSize = threadStack;
+    STATUS tooLargeStatus = STATUS_SUCCESS;
```

Comment:
(nit) Let's rename to something like "threadCreateStatus".

Reply:
Done